int32 ask_ns
---
int32 ns_result
